src/components/icons/CryptoIcon.tsx

```diff
@@ -101,7 +101,7 @@ const CryptoIconComponent = (props: Props) => {
 
   return (
     <View style={spacingStyle}>
-      {wallet == null ? null : <WalletSyncCircle size={size} wallet={wallet} />}
+      {wallet == null ? null : <WalletSyncCircle key={`${wallet.id}${String(tokenId)}`} size={size} wallet={wallet} />}
```

Comment:
I would add a comment to this key, so we remember why it's here:

`// Re-start the animation if the row gets recycled`